### ISSUE TITLE
[bug] [accessibility] screen reader does not read anything within the cosmic store

### ISSUE THREAD

**Issue by phoekjelle** [pop-os/cosmic-store#452]
When starting the screen reader, and hovering over the cosmic store, the screen reader doesnt tell me anything at all
